packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -1155,6 +1156,39 @@ void main() {
     expect(inputBox.hitTest(BoxHitTestResult(), position: inputBox.globalToLocal(newFourthPos)), isFalse);
   });
 
+  testWidgets('minLines cannot be greater than maxLines', (WidgetTester tester) async {
+    try {
+      await tester.pumpWidget(
+        overlay(
+          child: Container(
+            width: 300.0,
+            child: SelectableText(
+              'abcd',
+              minLines: 4,
+              maxLines: 3,
+            ),
+          ),
+        ),
+      );
+    } on AssertionError catch (e) {
+      expect(e.toString(), contains("minLines can't be greater than maxLines"));
+    }
```

Comment:
Add a line after this one to make sure the test fails if the assert is removed:
```dart
    fail('An assert should be triggered when minLines is greater than maxLines');
```

Reply:
You can check to see if it properly fails by removing the assert from your code and seeing if the test case fails